Tools/Scripts/libraries/webkitscmpy/webkitscmpy/test/setup_unittest.py

```diff
@@ -54,8 +54,6 @@ def test_github(self):
 
         self.assertEqual(
             captured.stdout.getvalue(),
-            'For detailed information about options this script is configuring see:\n'
-            'https://github.com/WebKit/WebKit/wiki/Git-Config#Configuration-Options\n'
```

Comment:
is this removal intentional?

Reply:
Yes. We moved this line so it's associated with a local repository, not all repositories (you can run `git-webkit setup` on a remote GitHub repository). I did this because this line doesn't make much sense in the context of a remote repository, where we really only need to setup a user-owned fork.